cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -32,38 +33,46 @@ import (
 type Reporter struct {
 	collector api_v2.CollectorServiceClient
 	logger    *zap.Logger
+	sanitizer zipkin2.Sanitizer
 }
 
 // NewReporter creates gRPC reporter.
 func NewReporter(conn *grpc.ClientConn, logger *zap.Logger) *Reporter {
+	zSanitizer := zipkin2.NewChainedSanitizer(
+		zipkin2.NewSpanDurationSanitizer(),
+		zipkin2.NewSpanStartTimeSanitizer(),
+		zipkin2.NewParentIDSanitizer(),
+		zipkin2.NewErrorTagSanitizer(),
```

Comment:
damn, just noticed that we internally didn't use all 4:
```go
func (config *Config) NewZipkinSanitizer(metrics metrics.Factory, logger *zap.Logger) zs.Sanitizer {
        sanitizers := zs.NewChainedSanitizer(
                zs.NewSpanDurationSanitizer(),
                zs.NewParentIDSanitizer(),
        )
        return &sanitizers
}
```
Can we create `zipkin2.StandardSanitizers` array instead, and then here only do this?
```go
zSanitizer := zipkin2.NewChainedSanitizer(zipkin2.StandardSanitizers...)`
```